azure-devops/checkout-sources.yml

```diff
@@ -64,5 +68,6 @@ steps:
     )
 
     git fetch --filter=tree:0 --depth=1 boostorg $(${{ parameters.boostMathSHAVar }})
-    git checkout -f FETCH_HEAD
+    git reset --quiet --hard FETCH_HEAD
+    git clean --quiet -x -d -f

```

Comment:
Does this deserve a `# Yes, just one "-f"; our submodules don't have submodules` comment? To be fair, I probably wouldn't notice the anomaly if all three of the changes in this PR weren't highlighted in a PR.

Reply:
That would be less distracting than a comment, even if it is unnecessary.